### PR TITLE
[JENKINS-50790] Temporarily override pageLoadTimeout for available plugins

### PR DESCRIPTION
**Pull request by raul-arabaolaza**
[JENKINS-50790](https://issues.jenkins-ci.org/browse/JENKINS-50790)

This should fix most of GitPluginTest and some others that are failing due to the
available plugins taking too much time to load on ci.jenkins.io

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java [hunk 1]
```diff
@@ -227,8 +228,12 @@ public boolean installPlugins(final PluginSpec... specs) throws UnableToResolveD
                 }
             }
         } else {
+            // JENKINS-50790 It seems that this page takes too much time to load when running in the new ci.jenkins.io
+            driver.manage().timeouts().pageLoadTimeout(time.seconds(60), TimeUnit.MILLISECONDS);
             visit("available");
-
+            driver.manage().timeouts().pageLoadTimeout(time.seconds(FallbackConfig.PAGE_LOAD_TIMEOUT), TimeUnit.MILLISECONDS);
+            // End JENKINS-50790
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/test/acceptance/po/PluginManager.java]: nit: `try-finally` would make this more explicit.

File: src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java [hunk 2]
```diff
@@ -80,6 +80,8 @@
 
     public static final String DOM_MAX_SCRIPT_RUN_TIME = "dom.max_script_run_time";
     public static final String DOM_MAX_CHROME_SCRIPT_RUN_TIME = "dom.max_chrome_script_run_time";
+    public static int PAGE_LOAD_TIMEOUT = 30;
+    public static int IMPLICIT_WAIT_TIMEOUT = 1;
```
Comment (olivergondza) [inline src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java]: Is not non-`final` for a reason?

**Comment by raul-arabaolaza** [comment]
@olivergondza Feedback addressed, let's see how this goes on the PR Builder

**Comment by raul-arabaolaza** [comment]
Ready for review

**Comment by raul-arabaolaza** [comment]
@reviewbybees done

@olivergondza It seems we are back to "only" 21 failing tests and this is worthy to be merged

**Comment by recena** [comment]
FYI, this seems to be happening only in `ci.jenkins.io`, in my environment works fine.

**Comment by olivergondza** [comment]
Merging this despite there are still some tests failing waiting to plugin manager page to load:

- https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-428/3/testReport/plugins/KerberosSsoTest/kerberosTicket/
- https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-428/3/testReport/plugins/JobDslPluginTest/should_ignore_removed_config_files/
- https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-428/3/testReport/plugins/JobDslPluginTest/should_ignore_removed_config_files/
- https://ci.jenkins.io/job/Core/job/acceptance-test-harness/job/PR-428/3/testReport/plugins/JobDslPluginTest/should_use_additional_classpath/

though I agree this is much better. While at it I am fixing the screen recording that could help us in investigating what is going on in https://github.com/jenkinsci/acceptance-test-harness/commit/d8dc8adea40ec446bc1a5565454401e591c07889. Thanks!

**Comment by raul-arabaolaza** [comment]
Yeah, we need to find the root cause and fix this as we can not be just increasing timeouts indefinitely, this is just a quick fix

**Comment by jglick** [comment]
https://github.com/jenkinsci/jenkins/pull/3428 _may_ help this.

**Pull request merged**